playbooks/vagrant.yml

```diff
@@ -0,0 +1,20 @@
+- include: python.yml
+
+- name: Prepare the vagrant VM for Ansible
+# =========================================
+  hosts: streisand-host
+  remote_user: "root"
+  become: true
+
+  tasks:
+    # Ansible uses `ip -4 route get 8.8.8.8` to set the `ansible_default_ipv4`
```

Comment:
Thanks! The workaround _seems_ stable on my machine, hopefully it won't bite us in the butt down the road.